.baseline/copyright/999_palantir.txt

```diff
@@ -0,0 +1 @@
+Copyright ${today.year} Palantir Technologies, Inc. All rights reserved.
```

Comment:
this file shouldn't have been added =/ 

Excavator should send a PR soon removing it

Reply:
removed in #231